bundles/org.openhab.binding.shelly/README.md

```diff
@@ -285,14 +285,26 @@ Values 1-4 are selecting the corresponding favorite id in the Shelly App, 0 mean
-| UNKNOWN        | Indicates that the status is currently unknown, which must not show a problem. Usually the Thing stays in this status when the device is in sleep mode. Once the device is reachable and was initialized the Thing switches to status ONLINE.                                                                |
-| ONLINE         | ONLINE indicates that the device can be accessed and is responding properly. Battery powered devices also stay ONLINE when in sleep mode. The binding has an integrated watchdog timer supervising the device, see below. The Thing switches to status OFFLINE when some type of communication error occurs. |
-| OFFLINE        | Communication with the device failed. Check the Thing status in the UI and openHAB's log for an indication of the error. Try restarting OH or deleting and re-discovering the Thing. You could also post to the community thread if the problem persists.                                                    |
-| CONFIG PENDING | The thing has been initialized, but device initialization is in progress or pending (e.g. waiting for device wake-up)                                                                                                                                                                                        |
-| ERROR: COMM    | Communication with the device has reported an error, check detailed status.                                                                                                                                                                                                                                  |
+| Status                | Description                                                                                                                  |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| INITIALIZING          | This is the default status while initializing the Thing.
+Once the initialization is triggered the Thing switches to Status ONLINE, CONFIGURATION_PENDING.                                                       |
+| UNKNOWN               | Indicates that the status is currently unknown, which must not show a problem.
+Once the device is reachable and was initialized the Thing switches to status ONLINE.                                                                  |
+| CONFIGURATION_PENDING | The thing has been initialized, but device initialization is in progress or pending (e.g. waiting for device wake-up).       |
+| ONLINE                | ONLINE indicates that the device can be accessed and is responding properly.
+Once initialized battery powered devices also stay ONLINE when in sleep mode.
```

Comment:
This appears in the status column when the Markdown is rendered, which is probably not intended?

Reply:
fixed